quickstarts/gatsby-build/config.yml

```diff
@@ -23,6 +23,6 @@ documentation:
       Ship performance data from Gatsby builds using OpenTelemetry
     url: https://newrelic.com/blog/best-practices/optimize-gatsby-build-opentelemetry
 keywords:
```

Comment:
It looks like our CI doesn't handle an empty `keywords` list. I'm going to remove this to see if it passes our checks.
```suggestion
```